lib/cs_guide_web/templates/venue/show.html.eex

```diff
@@ -47,6 +47,10 @@
   <section class="tc center pv5 w-90 w-100-ns">
     <h1 class="roboto-slab f2 lh2 dib">Drinks at <%= @venue.venue_name %></h1>
     <div class="flex-ns flex-wrap justify-center pv4-ns">
+      <!-- Cocktail card -->
+      <%= if @venue.num_cocktails && @venue.num_cocktails != 0 do %>
```

Comment:
Could this condition be simplified to `<%= if @venue.num_cocktails != 0 do %>`?

Reply:
@Cleop yes this should work when `num_cocktails` is `nil` as nil is different to 0.
It's a good point that we need to check the default value, we need to make sure the default value is nil and not 0

Reply:
ok, yes in this case as nil is different to 0 the card will be displayed. An enhancement would be to define the default value in Postgres to 0 then we could have `<%= if @venue.num_cocktails > 0 do %>`